error redefining macro 'wxID_OSX_*` when compiling

When compiling I receive the following error:
Erlang: 25.0.2
OS: Archlinux

```
wings_frame.erl:40:9: redefining macro 'wxID_OSX_HIDE'
%   40| -define(wxID_OSX_HIDE, 5250).
%     |         ^

wings_frame.erl:41:9: redefining macro 'wxID_OSX_HIDEOTHERS'
%   41| -define(wxID_OSX_HIDEOTHERS, 5251).
%     |         ^

wings_frame.erl:42:9: redefining macro 'wxID_OSX_SHOWALL'
%   42| -define(wxID_OSX_SHOWALL, 5252).
%     |         ^

wings_frame.erl:43:9: redefining macro 'wxID_OSX_MENU_LAST'
%   43| -define(wxID_OSX_MENU_LAST, 5255).
%     |         ^

make[3]: *** [Makefile:176: ../ebin/wings_frame.beam] Error 1
```

--- a/src/wings_frame.erl
+++ b/src/wings_frame.erl
@@ -37,10 +37,18 @@
 -define(IS_SPLIT(WinProp), (element(1, WinProp) =:= split
 			    orelse element(1, WinProp) =:= split_rev)).
 
+-ifndef(wxID_OSX_HIDE).
 -define(wxID_OSX_HIDE, 5250).
+-endif.
+-ifndef(wxID_OSX_HIDEOTHERS).
 -define(wxID_OSX_HIDEOTHERS, 5251).
+-endif.
+-ifndef(wxID_OSX_SHOWALL).
 -define(wxID_OSX_SHOWALL, 5252).
+-endif.
+-ifndef(wxID_OSX_MENU_LAST).
 -define(wxID_OSX_MENU_LAST, 5255).
+-endif.
 
 %% API  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 

